src/Microsoft.AspNet.OData.Shared/Formatter/Serialization/ODataResourceSerializer.cs

```diff
@@ -461,6 +461,29 @@ public virtual string CreateETag(ResourceContext resourceContext)
             return null;
         }
 
+        /// <summary>
+        /// Creates the <see cref="ODataNestedResourceInfo"/> to be written while writing this delta nested property (complex or entity).
+        /// </summary>
+        /// <param name="property">The property for which the nested resource info is being created.</param>
+        /// <param name="resourceContext">The context for the property instance being written.</param>
+        /// <returns>The nested resource info to be written. Returns 'null' will omit this property serialization.</returns>
+        /// <remarks>It enables customer to get more control by overriding this method. </remarks>
+        public virtual ODataNestedResourceInfo CreateDeltaNestedResourceInfo(IEdmProperty property, ResourceContext resourceContext)
```

Comment:
Why do we use `Delta` in the name of the method? Why not just `CreateNestedResourceInfo`?

Reply:
I see that we have another `CreateDynamicComplexNestedResourceInfo` method. Are there other scenarios of nested resource info that we would want to allow the customer to override?